test/fuzz/gen.ml

```diff
@@ -44,7 +45,8 @@ let expr_available_1_i32 if_else expr ~locals ~stack env =
   ]
   @ B.local_set_i32 env @ B.local_tee_i32 env @ B.global_set_i32 env
   @ if B.memory_exists env then B.memory_grow :: load_instr else []
-  @ B.expr_br_if env stack
+  @ B.expr_br_if env stack true
+  @ B.expr_br_if env stack false
```

Comment:
You should be able to call this only once and make it generate everything that is needed.

Reply:
For the moment, I still have the 2 separate calls.
After `expr_br_if refactoring`, it seems ok to keep 2 separate calls ... ?